### DISCUSSION TITLE
Wow, big fan of Dune looking forward to playing this!

### DISCUSSION THREAD

**Discussion by ErikDeBruijn** [stefanhendriks/Dune-II---The-Maker#309] [General]
Stefan, amazing what you've done. It already looks so good. The nostalgia...

I'll have a look if it plays on OS X this weekend! Otherwise we'll find a windows machine.

**Comment by stefanhendriks** [comment]
Hi @ErikDeBruijn , thanks! Its a Windows binary which seems to run fine via Parallels. Wine worked a few years back.. dunno about now ;-).

Making it cross-platform is on the wishlist, but requires quite some refactoring. Nothing new there, just needs time.

**Reply by ErikDeBruijn** [reply]
Totally understand that you can't do everything at once. :)

Good to know Wine is something to try. Once I've tried it (and think to do so), I'll let you know how it works.

**Reply by stefanhendriks** [reply]
yes Wine or Parallels. Although I believe Wine needs to run under 32-bit mode which can be challenging nowadays under Mac OS.

**Comment by arkahys** [comment]
hey , Nostalgy ! many thanks. I'll try to help a bit.

**Reply by stefanhendriks** [reply]
thanks!
